### PR TITLE
Central system polish for long election content

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Task: #5247 
Fixes: https://github.com/votingworks/vxsuite/issues/5236

An assortment of polish for issues found using an election definition with long content (e.g. 100 character contest and candidate names).

## Demo Video or Screenshot

Added to individual commits

## Testing Plan
Manual testing

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/ui/src/election_info_bar.tsx
Comment (jonahkagan) [inline libs/ui/src/election_info_bar.tsx:1]: Screenshot-VxAdmin-2024-08-21T16:51:19 753Z

Note that this change is not needed in the horizontal info bar on the lock screen
Screenshot-VxAdmin-2024-08-21T16:51:28 680Z

File: libs/ui/src/diagnostics/components.ts
Comment (jonahkagan) [inline libs/ui/src/diagnostics/components.ts:1]: Screenshot-VxCentralScan-2024-08-21T16:54:07 496Z

File: apps/design/frontend/src/ballot_screen.tsx
Comment (jonahkagan) [inline apps/design/frontend/src/ballot_screen.tsx:1]: Screenshot 2024-08-21 at 9 57 54 AM

File: apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx
Comment (jonahkagan) [inline apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx:1]: Screenshot-VxAdmin-2024-08-21T17:02:01 133Z
Screenshot-VxAdmin-2024-08-21T17:02:25 608Z

File: libs/ui/src/search_select.tsx
Comment (jonahkagan) [inline libs/ui/src/search_select.tsx:1]: https://github.com/user-attachments/assets/e689a4d0-4edf-4c9a-ae10-da42b7453842

File: apps/admin/frontend/src/components/reporting/filter_editor.tsx
Comment (jonahkagan) [inline apps/admin/frontend/src/components/reporting/filter_editor.tsx:1]: Screenshot-VxAdmin-2024-08-21T17:19:16 526Z

File: apps/central-scan/frontend/README.md [hunk 1]
```diff
@@ -15,14 +15,9 @@ pnpm start
...
 
-To set the election configuration you will either need to scan a smartcard (you
-can use `./scripts/mock-card` in [`libs/auth`](../../../libs/auth) for this),
-load an election.json file, or load an election package from
-[election-manager](../election-manager). You should load an election package if
-you intend to test hand-marked paper ballots (HMPBs).
-
-To display a batch of scanned ballots use the `MOCK_SCANNER_FILES` environment
-variable set as described in [`services/scan`](../../../services/scan).
+To scanned ballots without scanner hardware use the `MOCK_SCANNER_FILES`
```
Comment (carolinemodic) [inline apps/central-scan/frontend/README.md:18]: Nit: To scan*

**Review by carolinemodic — APPROVED** [review]

File: apps/central-scan/frontend/README.md (hunk 1 above)
Comment (jonahkagan) [reply apps/central-scan/frontend/README.md]: thanks!

**Pull request merged**